src/containers/FactoryLoader.tsx

```diff
@@ -223,15 +229,16 @@ export class FactoryLoaderContainer extends React.PureComponent<Props, State> {
     try {
       await this.props.requestFactoryResolver(location);
     } catch (e) {
-      this.showAlert('Failed to resolve a devfile.');
-      return undefined;
+      if (!isOAuthResponse(e)) {
+        this.showAlert('Failed to resolve a devfile.');
```

Comment:
is it possible to extract the error message from `e` and append it to the message?

Reply:
sure, I'll add the fixup